internal/component/prometheus/scrape/scrape.go

```diff
@@ -317,9 +317,9 @@ func New(o component.Options, args Arguments) (*Component, error) {
 		HTTPClientOptions: []config_util.HTTPClientOption{
 			config_util.WithDialContextFunc(httpData.DialFunc),
 		},
+		// NOTE: Native histograms are now always enabled in Prometheus v3.8.0+
```

Comment:
TODO: check this is safe and deprecate the setting in docs, make it no-op if so.